scripts/reborn_webui_v2_live_qa/test_run_live_qa.py

```
@@ -5860,6 +5885,13 @@ def __init__(self, status_code: int, body: dict[str, object]):
             def json(self):
                 return self._body
 
+            def raise_for_status(self):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `raise_for_status`

Add return type annotation: `None`

(ANN202)